lib/pymedphys/docs/contributing/index.md

```diff
@@ -0,0 +1,18 @@
+# Contributing
+
+This document is a work in progress.
```

Comment:
maybe this should have the reference to the individual platform guides?
And... documentation is a contribution that should be highlighted.
Not all physicists can code.  But those who *use* PyMedPhys can probably offer some documentation to address an area they banged their head on if the available documentation didn't cover their scenario...

Reply:
Yup, for a future merge though. With swapping to Jupyter Book we now have the following feature baked in quite nicely:

![image](https://user-images.githubusercontent.com/6559099/100303489-b4635f80-2ff0-11eb-8196-2b293c24a28c.png)

Which takes users to the GitHub online file edit interface, automatically creates a social fork, and pull request :).

Reply:
And that should get pointed out in the Contributing section.  In a future PR